returnn/torch/data/pipeline.py

```diff
@@ -306,7 +307,7 @@ def create_data_loader_from_batches(
     if loader_opts is None:
         loader_opts: Dict[str, Any] = {}
 
-    if loader_opts.get("num_workers"):
+    if loader_opts.get("num_workers", 0) > 0:
```

Comment:
Why this change?

Reply:
If for some reason num_workers is set, but set to 0.

Reply:
Ah, 0 is falsey. Never mind.

Reply:
Well but it's slightly different. E.g. when `num_workers` is set to a string, this crashes now. Or when `num_workers` is negative, it would not apply the logic anymore now. However, I specifically wanted that it would also apply the logic in those cases. Who knows what `DataLoader` might support in the future for `num_workers`. If it is some invalid value for `num_workers`, this is up to `DataLoader` to handle. RETURNN should mostly ignore it.

Reply:
I see, then I'll revert.